bin/check_required_files_present

```diff
@@ -1,64 +1,64 @@
 #!/usr/bin/env bash
-MISSING_FILES=0
-SHOW_PROGRESS='false'
+missing_files=0
+show_progress='false'
 
-if [ ! -d exercises ] ; then
-	echo "\"exercises\" directory not found, make sure you're running from the repository root" >&2
-	exit 1
+if [[ ! -d exercises ]] ; then
+    echo '"exercises" directory not found, make sure you are running from the repository root' >&2
+    exit 1
 fi
 
 function show_progress {
```

Comment:
Not comfortable with a `show_progress` function and a `show_progress` variable. Rename the variable?

Reply:
I'll wait on the other PRs and rebasing.

Reply:
Renamed.